Documentation/compatibility/wpf-accessibility-improvements.MD

```diff
@@ -0,0 +1,101 @@
+## 195: Accessibility improvements in WPF
+
+### Scope
+Major
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+__High Contrast improvements__
+
+- Focus for expander was not visible.  This is now fixed.
+- The text in CheckBox and RadioButton is difficult to see when selected.  This issue is now fixed.
+- The border of a disabled ComboBox is now the same color as the disabled text.
+- Disabled and focused buttons used the incorrect theme color.  This is now fixed.
+- Setting a ComboBox’s style to Toolbar.ComboBoxStyleKey caused the dropdown button to be invisible.  This is now fixed.
+- The sort indicator arrow in DataGrid used the incorrect theme colors, making it confusing.  This is now fixed.
+- The default hyperlink style changed to incorrect color on mouse over in high contrast themes.  This is now fixed.
+- Keyboard focus on radio buttons in high contrast themes is not visible.  This is now fixed.
+- DataGrid checkbox column does not use the expected colors for keyboard focus feedback.  This is now fixed.
+- Keyboard focus visuals are not visible on ComboBoxes and ListBoxes in high constrast themes.  This is now fixed.
```

Comment:
Do these as links --> `T:System.Windows.Controls.ComboBox` and `T:System.Windows.Controls.ListBox` controls